source/agora/consensus/data/Params.d

```diff
@@ -54,6 +54,12 @@ public immutable class ConsensusParams
     /// The address of commons budget
     public PublicKey CommonsBudgetAddress;
 
+    /// The maximum size of transaction data
+    public uint MaxSizeTransactionData;
```

Comment:
In Bitcoin, there is OP_RETURN.
The maximum value is 83 Bytes.
Bitcoin Cash has a maximum value of 200Byte

Reply:
See) https://docs.cryptoapis.io/rest-apis/blockchain-as-a-service-apis/btc/index#btc-transactions-creating-transactions